api/test/wfl/integration/modules/copyfile_test.clj

```diff
@@ -94,3 +89,55 @@
                   (update :items (partial map #(assoc % :options {})))
                   workloads/create-workload!
                   :workflows))))
+
+(defn mock-batch-update-workflow-statuses!
+  [tx {:keys [workflows items] :as _workload}]
+  (letfn [(update! [{:keys [id]}]
+            (jdbc/update! tx items
+                          {:status "Succeeded" :updated (OffsetDateTime/now)}
+                          ["id = ?" id]))]
+    (run! update! workflows)))
+
+(deftest test-workload-state-transition
+  (with-redefs-fn
+    {#'cromwell/submit-workflow                 (constantly (UUID/randomUUID))
```

Comment:
Why `constantly`?

Reply:
Take any number of inputs and return a UIUD. Ah but it'll be the same UUID. OK I'll fix.